File: 백준/Bronze/9713. Sum of Odd Sequence/Sum of Odd Sequence.c
#include<stdio.h>
int main(void){
    int n;
    scanf("%d",&n);
    int arr[100];

    for(int i = 0; i < n; i++){
        scanf("%d",&arr[i]);
        int sum = 0;
        for(int j = 1; j <= arr[i]; j++){
            if(j % 2 == 1)
                sum += j;
        }
        printf("%d\n",sum);
    }
    return 0;
}

File: 백준/Bronze/9713. Sum of Odd Sequence/Sum of Odd Sequence.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int [n];

        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
            int sum = 0;
            for(int j = 1; j <= arr[i]; j+=2){
                sum += j;
            }
            System.out.println(sum);
        }   
    }
}